sound/soc/sof/ipc4-topology.h

```diff
@@ -82,6 +84,26 @@ struct sof_ipc4_pipeline {
 	struct sof_ipc4_msg msg;
 };
 
+/**
+ * struct sof_ipc4_multi_pipeline_info - multi pipeline trigger IPC info
+ * @pipelines: Array of pipeline widgets to be triggered
+ * @count: number of pipelines to be triggered
+ */
+struct sof_ipc4_multi_pipeline_info {
+	struct snd_sof_widget *pipelines[SOF_IPC4_TRIGGER_MAX_PIPELINE_COUNT];
+	uint32_t count;
+};
+
+/**
+ * struct sof_ipc4_multi_pipeline_data - multi pipeline trigger IPC data
+ * @count: Number of pipelines to be triggered
+ * @pipeline_ids: Arrays of IDs of the pipelines to be triggered
+ */
+struct ipc4_pipeline_set_state_data {
+	uint32_t count;
+	uint32_t pipeline_ids[SOF_IPC4_TRIGGER_MAX_PIPELINE_COUNT];
```

Comment:
struct sof_ipc4_multi_pipeline_info  is very similar to struct ipc4_pipeline_set_state_data.  can we use the same order for "uint32_t count" ?